pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -550,9 +554,12 @@ func (c *Cluster) initClusterInfo(cephVersion cephver.CephVersion, clusterName s
 
 	k := keyring.GetSecretStore(c.context, c.ClusterInfo, c.ownerInfo)
 	// store the keyring which all mons share
-	if _, err := k.CreateOrUpdate(keyringStoreName, c.genMonSharedKeyring()); err != nil {
+	secretRef, err := k.CreateOrUpdate(keyringStoreName, c.genMonSharedKeyring())
```

Comment:
This value looks like the resource version, not a reference to the secret, right? If so, a different var name would help understand this better. 
```suggestion
	secretResourceVersion, err := k.CreateOrUpdate(keyringStoreName, c.genMonSharedKeyring())
```